content/en/apps/guides/performance/replication.md

```diff
@@ -22,6 +22,8 @@ The most common restriction is by place. This is where we check the user's `faci
 
 For example, if a CHP's `facility_id` property is set to the ID of the Maori Hill clinic, then they will be able to see all patients and all reports about patients at that clinic.
```

Comment:
@michaelkohn can we add here this?
> The `facility_id` is an array that contains one or many facility ids. Many facilities Ids will work only by enabling the `can_have_multiple_places` permission. You can use the Administration app to assign multiple facilities. 

@esthermmoturi Not sure if `array` is a good understandable word, or too technical.

Reply:
Would it be helpful to link one of the round ups call explaining this feature? I think video will make 100% times easier for people

Reply:
Thanks for the recommendations @latin-panda !  I've updated the text a bit and added links to the videos.